build/README.md

```diff
@@ -57,10 +57,17 @@ which may or may not work (and are not officially supported).
 
 ## Basic Process
 
-- Edit `build/builder/Dockerfile` as desired
-- Run `build/builder.sh init` to test -- this will build the image locally. Beware this can take a lot of time. The result of `init` is a docker image version which you can subsequently stick into the `version` variable inside the `builder.sh` script for testing locally.
-- Once you are happy with the result, run `build/builder.sh push` which pushes your image towards Docker hub, so that it becomes available to others. The result is again a version number, which you then *must* copy back into `builder.sh`. Then commit the change to both Dockerfile and `builder.sh` and submit a PR.
-- Finally, use this version number to update the `builder.dockerImage` configuration parameter in TeamCity under the [`Cockroach`](https://teamcity.cockroachdb.com/admin/editProject.html?projectId=Cockroach&tab=projectParams) and [`Internal`](https://teamcity.cockroachdb.com/admin/editProject.html?projectId=Internal&tab=projectParams) projects.
+- Edit `build/builder/Dockerfile` as desired and commit your changes.
```

Comment:
How about retaining some of the prior text describing how to do local development so folks can test things out locally? This is especially important for folks who can't run the new builder image build config.